m4_sessions/1811_inning/failure.md

```diff
@@ -25,6 +29,26 @@
 
 ## Session Outline (Agenda for session, include timing for each session and who is facilitating)
 
+Introductions, name, mod, something you are good at and how long it took you to actually become good at it 5 mins
+
+Short video  talking about how failure is a good thing 5 mins
+
+Redefinition of failure 7 mins. (ask how people would define failure, ask how people feel when they fail, ask what they think of others when they fail)
+
+Short video of successful people and their failures 5 min
+
+Discussion about how failure is not normalized and how if we knew about every time people had failed rather than only the successful stories that people would not feel so bad about it. 7min
 
+Each of us Share a story of failure and how it was positive 5mins
+
+Breakout groups Discussing their failures 7min
+
+Potentially another video or activity 5min
+
+Group discussion about how to encourage failure, overcome failure, encourage others to be more open about failure. 7min
+
+Conclusion and reflection. talk about how long we have been in turing and compare it to how much time it took us to become good at what we originally said at the beginning.  5 min.
```

Comment:
Maybe also...what are you leaving with?  How will you use this in your week 6?

Reply:
We're going to end with talking about failing is the normal process of writing code and connect our gear-up back with development in general